TextAnalyzer/CountEveryLetterReportCommand.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace TextAnalyzer
+{
+    class CountEveryLetterReportCommand : ICommand
+    {
+        public string Description => "Generate report for each letter from file";
+
+        public void Activate()
+        {
+            List<LetterOccurrenceModel> letterOccurrenceList = new List<LetterOccurrenceModel>();
+
+            String text = Regex.Replace(TextFileFetcher.GetTextFileString(), @"(\W|\d|_)+", "");
+
+            foreach (char letter in text)
+            {
+                LetterOccurrenceModel letterOccurrenceModel = letterOccurrenceList.Find(item => item.letter == letter);
+                if (letterOccurrenceModel == null)
+                {
+                    letterOccurrenceList.Add(new LetterOccurrenceModel(letter, 1));
+                }
+                else
+                {
+                    letterOccurrenceModel.Increase();
+                }
+            }
```

Comment:
Może warto się zastanowić czy byłoby możliwe przerobienie to na Dictionary gdzie kluczem byłaby litera a wartością twój `LetterOccurranceModel` tylko że już bez konieczności implementacji w nim pola na literę